code/game/objects/structures/vehicles/wheelchair.dm

```diff
@@ -207,7 +207,9 @@
 /obj/structure/bed/chair/vehicle/wheelchair/motorized/syndicate
 	nick = "medical malpractice"
 	desc = "A chair with fitted wheels which is powered by an internal cell. It seems to ride higher than other wheelchairs."
-	default_cell_path = /obj/item/weapon/cell/hyper
+
+/obj/structure/bed/chair/vehicle/wheelchair/motorized/syndicate/getMovementDelay()
+	return (..() + 1) //Somewhat slower
```

Comment:
Oh, and let's not forget that to have the right to talk about performance, you need to kill StonedMC and adopt the scheduler.